### ISSUE TITLE
addressable-resolver.yaml duplicates roles from knative eventing

### ISSUE THREAD

**Issue by grantr** [google/knative-gcp#1343]
**Describe the bug**
https://github.com/google/knative-gcp/blob/master/config/core/roles/addressable-resolvers-clusterrole.yaml contains some ClusterRole objects that are near duplicates of those defined in https://github.com/knative/eventing/blob/master/config/core/roles/addressable-resolvers-clusterrole.yaml.

For the most common case of installing both knative eventing and knative-gcp, this means some roles will conflict and their values will depend on which project was installed first.

 **Expected behavior**
Each role is installed only once.

 **To Reproduce**
1. Install eventing, then knative-gcp.
1. Observe that the `service-addressable-resolver` ClusterRole includes the `services/status` resource, which is only included in knative-gcp and not Knative eventing.
